lib/scarlet/server.rb

```diff
@@ -46,8 +48,10 @@ def initialize cfg
       @current_nick = config.nick
       config.control_char ||= Scarlet.config.control_char
       config.buffer_rate ||= 1
+      config.scheduler_rate ||= 1
       config.freeze
 
+      @reminder_scheduler = ReminderScheduler.new self
```

Comment:
tl;dr Rufus expects a block, the ReminderScheduler just takes `data`, I didn't remove it until we're sure it isn't used anymore. If you know how to serialize a ruby block, lemme know!

## Long Explanation
Nope, Rufus takes a block and executes it at the specified time or after the duration, the reminder scheduler takes a `Scarlet::ReminderScheduler::Task` structure, the scheduler loops every second and checks the tasks, any `completed` ones will be sent to the server